scripts/prebuild/index.ts

```diff
@@ -0,0 +1,186 @@
+import { ethers } from 'ethers';
+import fs from 'fs';
+import { resolve } from 'path';
+import keyBy from 'lodash/keyBy';
+import entries from 'lodash/entries';
+
+import { hashDataMap } from 'lib/multicall/hashData/hashDataMap';
+
+import {
+  OPEN_INTEREST_IN_TOKENS_KEY,
+  OPEN_INTEREST_KEY,
+  POOL_AMOUNT_KEY,
+  POSITION_IMPACT_POOL_AMOUNT_KEY,
+  SWAP_IMPACT_POOL_AMOUNT_KEY,
+} from "config/dataStore";
+
+import { DataStore__factory } from "typechain-types/factories/DataStore__factory";
+import { SyntheticsReader__factory } from "typechain-types/factories/SyntheticsReader__factory";
+
+const ARBITRUM = 42161;
+const AVALANCHE = 43114;
+const AVALANCHE_FUJI = 43113;
```

Comment:
`config/chains`

Reply:
Had to move it into `src/config/static/chains.ts`